assets/js/components/Dashboard/PlaceRow.tsx

```diff
@@ -152,6 +152,25 @@ const PlaceRow = (props: PlaceRowProps): JSX.Element => {
     )?.inlineMap;
   };
 
+  const formatStationName = (stationName: string) => {
+    let isTerminalStop = false;
+    if (props.filteredLine) {
+      const line = STATION_ORDER_BY_LINE[props.filteredLine.toLowerCase()];
+      const terminalStops = line.filter((line) => line.isTerminalStop);
+      isTerminalStop = terminalStops.some((stop) => stationName === stop.name);
+    } else {
+      isTerminalStop = Object.keys(STATION_ORDER_BY_LINE)
+        .filter((key) => !["silver", "mattapan"].includes(key))
+        .some((key) => {
+          const line = STATION_ORDER_BY_LINE[key];
+          const terminalStops = line.filter((line) => line.isTerminalStop);
+          return terminalStops.some((stop) => stationName === stop.name);
+        });
```

Comment:
Mind leaving in-line comments for some of this? The "why we do this" won't be clear down the road.